src/components/PageHeader/ObjectHome.js

```diff
@@ -4,7 +4,7 @@ import cx from 'classnames';
 import enhanceWithClickOutside from 'react-click-outside';
```

Comment:
I think the Menu has been made optional for `ObjectHome` recently, please check whether this change affects your PR and merge the conflicts accordingly.